celer/dropin_sklearn.py

```diff
@@ -105,6 +107,15 @@ class Lasso(Lasso_sklearn):
       http://proceedings.mlr.press/v80/massias18a.html
     """
 
+    _parameter_constraints: dict = {
```

Comment:
Another fix would be to define our own parameters constraints specific to the class.

With these changes, we pass the Lasso test.

Reply:
I think it's better to have the checks than to remove `_validate_data`